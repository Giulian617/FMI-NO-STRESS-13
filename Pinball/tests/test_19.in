100 15610
34 55 8
96 34 3
57 55 9
19 34 2
51 55 10
88 19 4
85 88 3
37 85 3
25 55 4
80 51 8
1 88 3
72 51 8
39 55 7
44 85 3
89 72 6
50 55 5
46 1 6
53 46 4
63 25 5
49 96 7
29 49 7
94 29 7
59 55 4
17 96 9
35 55 3
12 53 4
97 63 1
36 80 3
66 57 1
20 96 4
22 34 5
92 63 8
18 49 4
69 72 7
15 88 5
42 53 3
99 94 7
81 17 2
21 99 1
47 36 4
43 59 8
75 18 5
11 49 6
95 75 3
56 12 3
90 22 8
93 39 10
87 51 2
24 22 1
14 20 5
41 92 9
5 99 10
9 75 8
4 55 8
86 25 5
54 51 6
26 96 5
10 56 8
91 96 2
28 51 3
98 22 9
13 90 4
74 5 7
23 4 2
48 39 10
32 91 8
3 4 4
71 47 4
77 20 7
60 24 4
61 4 9
82 20 2
33 17 2
30 55 7
7 53 10
79 85 4
64 39 9
62 56 1
27 91 9
73 60 7
16 29 1
100 49 2
65 23 8
70 12 9
76 97 3
67 56 2
58 75 1
52 67 3
31 17 7
38 79 10
83 3 3
2 46 6
84 52 7
6 66 1
45 36 4
8 9 6
40 80 2
68 37 4
78 91 2
