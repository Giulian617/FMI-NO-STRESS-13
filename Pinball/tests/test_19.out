378758521
438088463
916037805
416190356
285514172
974292706
608241707
156036284
732141081
632884411
82793299
149827428
36786681
450750429
61894085
482797911
647246687
805815071
347743320
59753794
770836605
891111233
626579347
820639198
694362860
471195468
107244409
725132075
560358534
342942560
174594400
873630616
759645254
144106038
694362860
961279534
553335354
471195468
982558260
182711890
740207619
942526974
821229655
805815071
611917938
91322098
997890931
187502681
380082293
564179838
107244409
628148031
279819114
370697371
